README.md

```diff
@@ -50,18 +55,25 @@ $ rails spec
 > NOTE: You'll need to have docker and docker-compose installed
 
 Build the points docker image
+
 ```
 $ docker-compose build
 ```
 
 Setup database and run migrations
+
 ```
 $ docker-compose run web rails db:setup
 ```
 
 Run the app
+
 ```
-$ docker-compose up
+$ docker-compose up web
+
+or
+
+$ docker-compose up web-next
```

Comment:
There rest of the changes are just the code formatter fixing markdown code style